internal/jimm/auditlog/cleanup.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2025 Canonical.
+package auditlog
+
+import (
+	"context"
+	"time"
+
+	"github.com/juju/zaputil/zapctx"
+	"go.uber.org/zap"
+)
+
+// auditLogCleanupTime indicates that we poll at 9 AM.
+var auditLogCleanupTime = pollTimeOfDay{
+	Hours: 9,
+}
+
+// StartCleanup loop forever and checks daily for any logs
+// that need to be cleaned up. This method should be run
+// in a separate Go routine to avoid blocking, it will terminate
+// when the provided context is cancelled.
+func (j *auditLogManager) StartCleanup(ctx context.Context) {
```

Comment:
also, again, not a fan of splitting methods of a type in multiple files

Reply:
Yeah it's not ideal but I'm preferring to keep it like this to make things easier to find.